backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/managedblock/ManagedBlockStorageCommandUtil.java

```diff
@@ -142,7 +142,9 @@ public VDSReturnValue attachManagedBlockStorageDisk(ManagedBlockStorageDisk disk
 
```

Comment:
Change makes sense - but is required only to allow LSM which is not supported yet.
Can we defer this change so we can merge sooner the lightos support?

Until we do this change, all disks will use blank uuid. Adding the sd id is likely
wanted also in 4.5.1, but we can add it after the basics are merged, no?

Reply:
the vdsm PR included these changes so I tested them all together, I can separate it now since it's merged in vdsm